.fakeApi/routes.json

```diff
@@ -0,0 +1,5 @@
+{
+  "/api/v1/": "/",
+  "/instructors": "/instructors",
+  "/instructors/:slug": "/instructors/0"
```

Comment:
Need to figure out a way to map routes to match other environments; so the `0` here should actually be `:id` so `:slug` renders the instructor with id matching the slug